react/features/mobile/external-api/middleware.js

```diff
@@ -349,6 +352,25 @@ function _registerForNativeEvents(store) {
 
 }
 
+/**
+ * Unregister for events sent from the native side via NativeEventEmitter.
+ *
+ * @param {Store} store - The redux store.
```

Comment:
Leftover param from the register function

Reply:
fixed